backend/config.py

```diff
@@ -138,14 +143,22 @@ class Config(Loggable):
     analyte_output_units: str = MILLIGRAMS_PER_LITER
     waterlevel_output_units: str = FEET
 
-    use_csv: bool = True
-    use_geojson: bool = False
+    # use_csv: bool = True
+    # use_geojson: bool = False
 
-    def __init__(self, model=None, payload=None):
+    output_format: OutputFormat = OutputFormat.CSV
```

Comment:
In the GeoJSON implementation I called it site_file_type since the timeseries remains a csv. After this gets merged to pre-production I'll use the enum and will change the name to be specific to sites